invenio_app_ils/views.py

```diff
@@ -39,3 +39,12 @@ def success_response(self, user):
             )
         )
         return jsonify(user_payload), 200
+
+
+logged_out_blueprint = Blueprint('logged_out', __name__)
```

Comment:
This is great, but I am thinking that we could add an extra safe check and register this view **only** in dev env, WDYT? My fear is that this is registered and used on production envs, and we don't want that.

I would convert this to a function, similar to [here](https://github.com/inveniosoftware/invenio-app-ils/blob/master/invenio_app_ils/ill/views.py#L30):

Pseudocode below:
```
def create_root_blueprint(...):
   ....
   if current_app.config is dev:
       register_the_blueprint
       
   # otherwise do nothing
```